Exercise_A/index.html

```diff
@@ -6,10 +6,10 @@
     <title></title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
+    <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!-- CODE HERE -->
+    <a href="about.html"><img src="animation.jpg" alt="animation"></a>
 
     <script src="" async defer></script>
   </body>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!